stack

```diff
@@ -217,7 +217,7 @@ function solver() {
   load-local-env
   export WEB3_PRIVATE_KEY=${SOLVER_PRIVATE_KEY}
   export LOG_LEVEL=debug
-  go run . solver --network dev
+  go run . solver --network dev "$@"
```

Comment:
This adds on extra args from the command line. We had it on other services, but it was missing on the solver.